packages/react-api/__tests__/api/stats.test.js

```diff
@@ -135,6 +135,82 @@ describe('stats', () => {
       });
     });
 
+    test('big query source - should return stats using POST', async () => {
+      const hugeQuery = Array.from(Array(5000)).join(',');
+      const QUERY_TEST = {
+        input: {
+          source: { ...QUERY_SOURCE, data: hugeQuery },
+          column: 'injuries'
+        },
+        url: `https://gcp-us-east1.api.carto.com/v3/stats/carto-ps-bq-developers/injuries`,
+        output: TABLE_AND_QUERY_STATS
+      };
+
+      const fetchMock = (global.fetch = jest.fn().mockImplementation(async () => {
+        return {
+          ok: true,
+          json: async () => QUERY_TEST.output
+        };
+      }));
+
+      const abortController = new AbortController();
+
+      const res = await getStats({ ...QUERY_TEST.input, opts: { abortController } });
+
+      expect(res).toEqual(QUERY_TEST.output);
+
+      expect(fetchMock).toBeCalledWith(QUERY_TEST.url, {
+        method: 'POST',
```

Comment:
@zbigg Should we check the `method: 'GET'` in the previous tests?

Reply:
GET is default in `fetch` api, and this assertion is exact match, so we should be safe